### ISSUE TITLE
could not access KVM kernel module: Operation not permitted

### ISSUE THREAD

**Issue by atomiclucky** [mediabots/Linux-to-Windows-with-QEMU#2]
http://prntscr.com/nf1rp6 what's up with that, man? 
Do I need to install something additional?  I already changed the port and I'm sure that my vps host allows virtualization 
OS: UBUNTU 18.04 LTS (bionic) 
Kernel: 4.15.0

**Comment by mediabots** [comment]
https://i.imgur.com/RbJbFfH.png
I can see in your case my script return cores=0 , which should be > 0. And in the end their is an error "could not access KVM kernel module". That might happened if your hosting provider tricked in BIOS/Kernel.


Let me know the output of below command:
lscpu | grep CPU\(s\)


Right now,what you can do is, 
the long white coloured command right below "[ For Debugging purpose ]"
starts with /usr/bin/qemu-system-x86_64 and ends with  & disown %1
copy that text into some text editor & then change 'cores=0' to 'cores=1'
and run that new command on your VPS


And see, if its running successfully with out returning any error.


---
For conformation , I just ran my script on a Scaleway.com vps (paln - DEV1-M) : Ubuntu 18.04 | 4GB RAM | 3 Cores | 40 GB SSD
And it worked without any error
https://i.imgur.com/urRXsb4.png﻿

**Comment by atomiclucky** [comment]
I changed the cores in the command to 1 and ran it again. It gave the same error. Also the lscpu output:

CPU op-mode(s):        32-bit, 64-bit
CPU(s):                0
CPU family:            6
Model name:            Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz
CPU MHz:               883.618

**Comment by atomiclucky** [comment]
rip me gonna try openvz

**Comment by mediabots** [comment]
You can also try VPS from https://www.scaleway.com/en/pricing/ or droplets from https://www.digitalocean.com/pricing/

Their VPS are KVM enabled and I tested both of them and its working

Even you can buy a cheap dedicated server from here :
https://oneprovider.com/dedicated-servers/1gbps , prices start at $9 to $10

Regards

**Comment by atomiclucky** [comment]
are you affiliated with them or what, because on my local provider you can get 8gb RAM  2,2ghz * 6 and 70 gb ssd for 7.38 Euro. That's a pretty good price and I'm not changing

**Comment by mediabots** [comment]
Do I look like that those big Hosting companies would hire me (a piece of coder) for promoting them?
They are biggest companies in web hosting world, they don't recommendation from a start up coder.

Anyway , did I posted any referral links? from my last post urls ,clearly I did not.

I have a topic about collection of Cheapest Dedicated server in online https://www.blackhatworld.com/seo/list-cheapest-dedicated-servers-ssd-sata-1-gbit-port-windows-linux-os.1105442/

None of those listing have any referral links

So from next time before accusing some one , use some logic

And I must admit your local server guy providing good rates (though disk space wise 70 GB SSD is low,but other configs are good)
